edgelet/edgelet-http/src/certificate_manager.rs

```diff
@@ -142,11 +143,9 @@ impl<C: CreateCertificate + Clone> CertificateManager<C> {
     }
 
     fn create_cert(&self) -> Result<Certificate, Error> {
-        // In some use cases, the CA cert might change - to protect against that,
-        // we will retry once (after attempting to delete) if the cert creation fails.
-        let cert = if let Ok(val) = self.crypto.create_certificate(&self.props) {
-            val
-        } else {
+        // remove any certificate with this alias (if it exists),
+        // then create the new certificate.
+        let cert = {
```

Comment:
I think we can move assign operation to the line below let cert = self.crypto.create_certificate(..)